style.css

```diff
@@ -0,0 +1,102 @@
+
+body {
+  font-family: "Poppins", sans-serif;
+  padding: 0%;
+  margin: 0;
+}
+
+.first-sector {
+  height: auto;
+  width: 100%;
+  min-height: 100vh;
+  border-radius: 0.25rem;
+}
+
+.main {
+  padding: 0 20px;
+}
+
+.header-comtainer {
+  display: flex;
+  justify-content: space-between;
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓